modelservice/master.py

```diff
@@ -90,8 +90,31 @@ def BuildModels(self, request: BuildModelsRequest, context) -> BuildModelsRespon
         )
 
     def GetModel(self, request: GetModelRequest, context) -> GetModelResponse:
-        info(f"Received request to retrieve model(s)")
-        return GetModelResponse()
+        info(f"Received request to retrieve model")
+
+        # job_id: str = request.model_id
+        gis_join: str = request.gis_joins
```

Comment:
Can you update this to `request.gis_join` (singular) in the proto file and regenerate the protobuf/grpc files

Reply:
Updated